src/main/java/gov/usgs/owi/nldi/controllers/NetworkController.java

```diff
@@ -26,6 +26,9 @@
 @RequestMapping(value="linked-data/comid/{comid}/navigate/{navigationMode}")
 public class NetworkController extends BaseController {
 
+	//swagger documentation for /linked-data/{featureSource}/{featureID}/navigate/{navigationMode} endpoint
+	@Operation(summary = "getFlowlines", description = "returns the flowlines for the specified navigation in WGS84 lat/lon GeoJSON")
```

Comment:
Sames changes for this file as for LookupController.java.  It needs and import statement and the annotation needs to be below the constructor and above the getFlowlines method (around line 38).